nss-day-cohort-46/reactive-nutshell-gauntlets-of-adventure-1
# Tasks modules created w/ basic functionality

feat: created TaskProvider TaskList TaskCard TaskForm Tasks.css / data provider functional / TaskList is rendering buttons and tasks to DOM / TaskCard renders individual tasks / TaskForm renders when add Task is clicked

# Description
- created dataProvider and included methods for getTasks, getTasksById, addTask, deleteTask
- created basic form for name and expectedCompletion date. //need to add <label&input> for taskDetails.
- individual tasks are rendering to DOM and have buttons for 'completed' and 'delete' //need to finish button functionality

## Type of change

Please delete options that are not relevant.


# Testing Instructions
git fetch --all
git checkout nah-createTasks
make sure your server is running (npm start)
use login to gain access to application
click on tasks on header
click on add Task and fill out the form / click on create Task
page renders back to Tasks with new task on DOM

// delete and complete are NOT functional at this time

# Checklist:

        - src/components/auth/Login.js error is still present
